src/checkers/inference/InferenceMain.java

```diff
@@ -154,8 +154,7 @@ private void startCheckerFramework() {
                 "-Xmaxwarns", "1000",
                 "-Xmaxerrs", "1000",
                 "-XDignore.symbol.file",
-                "-AprintErrorStack",
-                "-Awarns"));
```

Comment:
What Is the reason of removing "-Awarns"?

Reply:
Definition of "-Awarns" is here:
    // Treat checker errors as warnings
    // org.checkerframework.framework.source.SourceChecker.report
    "warns",

>     // Treat checker errors as warnings
    // org.checkerframework.framework.source.SourceChecker.report
    "warns",
Inference, it suppresses error of unsatisfiable constraint over two ConstantSlots into warning, and let compiler pass. As a result, we don't see the error message and inference doesn't exit directly.
After I removed this, I see the error message and inference exits directly when encountered unsatisfiable constraints.

Reply:
Ok, got it. Thanks for the explanation!